README.md

```diff
@@ -11,117 +11,47 @@ developed.](https://img.shields.io/badge/repo%20status-Active-green.svg)](https:
-Mainnet is currently running as `cosmoshub-4`. Archives of the state of `cosmoshub-1`, `cosmoshub-2`, and `cosmoshub-3` are available [here](./docs/resources/archives.md).
-
-If you are looking for historical genesis files and other data [`cosmos/mainnet`](http://github.com/cosmos/mainnet) is an excellent resource. Snapshots are also available at [cosmos.quicksync.io](https://cosmos.quicksync.io).
-
-<br/>
-
-## 🤝 — How to contribute
-
-___
+## Documentation
 
-Check out [contributing.md](CONTRIBUTING.md) for our guidelines & policies for how we develop the Cosmos Hub. Thank you to all those who have contributed!
+For the most up to date documentation please visit [hub.cosmos.network](https://hub.cosmos.network).
 
-<br/>
+For an overview of the Interchain Stack (the Cosmos SDK, IBC, etc.), as well as examples and exercises to help developers get a quick start, please visit [tutorials.cosmos.network](https://tutorials.cosmos.network).
 
-## 💬 — Talk to us
+### Additional resources
 
-___
+**For node operators:** If you're interested in running a node on the current Cosmos Hub, check out the docs to [Join the Cosmos Hub Mainnet](https://hub.cosmos.network/main/hub-tutorials/join-mainnet).
 
-We have active, helpful communities on Twitter, Discord, and Telegram.
+**For validators:** If you want to participate and help secure the Cosmos Hub and its [consumer chains](https://hub.cosmos.network/main/interchain-security), check out becoming a validator. Information on what a validator is and how to participate as one can be found at the [Validator FAQ](https://hub.cosmos.network/main/validators/validator-faq). If you're running a validator node on the Cosmos Hub, reach out to a Janitor on the [Cosmos Developers Discord](https://discord.gg/interchain) to join the `#cosmos-hub-validators-verified` channel.
 
-|    |    |
-| -- | -- |
-| Cosmos Developers Discord | <a href="https://discord.gg/cosmosnetwork"><img src="https://img.shields.io/badge/Discord-7289DA?style=for-the-badge&logo=discord&logoColor=white" alt="Discord" height="30"/></a> |
-| Cosmos Twitter | <a href="https://twitter.com/cosmos"><img src="https://img.shields.io/badge/Twitter-1DA1F2?style=for-the-badge&logo=twitter&logoColor=white" alt="Tweet" height="30"/></a> |
-| Cosmos Gov Twitter | <a href="https://twitter.com/CosmosGov"><img src="https://img.shields.io/badge/Twitter-FEFEFE?style=for-the-badge&logo=twitter&logoColor=blue" alt="Tweet" height="30"/></a> |
-| Cosmos Telegram | <a href="https://t.me/cosmosproject"><img src="https://img.shields.io/badge/Telegram-2CA5E0?style=for-the-badge&logo=telegram&logoColor=white" alt="Telegram" height="30"/></a> |
+**For delegators:** If you still want to participate on the Cosmos Hub, check out becoming a delegator. Information on what a delegator is and how to participate as one can be found at the [Delegator FAQ](https://hub.cosmos.network/main/delegators/delegator-faq).
```

Comment:
Not sure about the "still"? Maybe you could say something similar to: "If you want to secure the Cosmos Hub without running a validator, you can consider becoming a delegator."